frontend/src/views/Home.vue

```diff
@@ -9,10 +9,20 @@ import { TopFiveKey } from '@/components/home/TopFive.vue';
 
 const { result: topFiveResult, loading: topFiveLoading } = useQuery<{
```

Comment:
`loading` 값은 왠지 boolean 값이 나올것 같은데, `topFiveLoading`이 적절할까요?

[Bool 변수 이름 제대로 짓기 위한 최소한의 영어 문법](https://soojin.ro/blog/naming-boolean-variables) 블로그 글을 한번 참고해보시면 좋을 것 같네요

Reply:
코멘트 감사합니다! 보통 is를 붙혀서 짓긴하는데 라이브러리단에서 아예 loading으로 제공해버려서 같은 맥락으로 지었던거 같아요 아래와 같이 변경해볼게요 :)

```suggestion
const { result: topFiveResult, loading: isTopFiveLoading } = useQuery<{
```